web/groups/hlt.json

```
@@ -18,6 +18,12 @@
   "CAHitNtupletCUDA|hltPixelTracksGPU": "Pixels|GPU",
   "CAHitNtupletCUDA|hltPixelTracksHitQuadruplets": "Pixels|GPU",
   "CAHitNtupletCUDA|hltPixelTracksSoA*": "Pixels",
+  "CAHitNtupletCUDAPhase1|hltPixelTracksCPU": "Pixels",
+  "CAHitNtupletCUDAPhase1|hltPixelTracksCPUOnly": "Pixels",
+  "CAHitNtupletCUDAPhase1|hltPixelTracksCUDA": "Pixels|GPU",
+  "CAHitNtupletCUDAPhase1|hltPixelTracksGPU": "Pixels|GPU",
+  "CAHitNtupletCUDAPhase1|hltPixelTracksHitQuadruplets": "Pixels|GPU",
+  "CAHitNtupletCUDAPhase1|hltPixelTracksSoA*": "Pixels",
```

I think this should be split into
```suggestion
  "CAHitNtupletCUDAPhase1|*cpu": "Pixels",
  "CAHitNtupletCUDAPhase1|*cuda": "Pixels|GPU",
```

---

Changed as suggested.